draft-ietf-wpack-bundled-responses.md

```diff
@@ -85,8 +85,10 @@ Bundle parsers support two primary operations:
 ## Naming a representation {#semantics-naming}
 
 Representations within a bundle are named by their `Content-Location` (Section
-8.7 of {{!I-D.ietf-httpbis-semantics}}), which holds a URL. This is also known
-as the representation's URL.
+8.7 of {{!I-D.ietf-httpbis-semantics}}). This is also known as the
+representation's URL. It is either an absolute-URI or a partial-URI (Section
+2.7 of {{!RFC7230}}). In the latter case, the referenced URL is relative to the
+bundle's URL unless a section in the bundle specifies a different base URL.
```

Comment:
This line would be clearer if it specified that it is not talking about any of the default sections defined in the spec, but about an additional one defined by an extension.

I am not sure about the best way to express this idea, so take this as a suggestion:

```suggestion
bundle's URL unless an additional section in the bundle specifies a different
base URL.
```

Reply:
Done.